pkg/server/server.go

```diff
@@ -696,9 +696,15 @@ func (s *ImmuServer) UpdateMTLSConfig(ctx context.Context, req *schema.MTLSConfi
 	return nil, ErrNotSupported
 }
 
+// ServerInfo returns health and status information about the server instance.
```

Comment:
Please add / move the comment into the `.proto` file - that way it will propagate to generated code too.